miniwin/CMakeLists.txt

```diff
@@ -52,6 +52,26 @@ else()
   message(STATUS "🧩 OpenGL ES 2.x support not enabled")
 endif()
 
+if(NINTENDO_3DS)
+  find_library(CITRO3D_LIBRARY NAMES citro3dd)
+  find_library(CTRU_LIBRARY NAMES ctru)
```

Comment:
Can't you use citro3d unconditionally on nintendo 3ds?

Reply:
citro3d and libctru are two separate libraries.

Oh, `citro3dd` is the debug build. Maybe with -DISLE_DEBUG we can switch the one we want to use